unstructured/partition/pdf_image/pdfminer_utils.py

```diff
@@ -34,6 +34,19 @@ def extract_image_objects(parent_object: LTItem) -> List[LTImage]:
     return objects
 
 
+def extract_text_objects(parent_object: LTItem) -> List[LTTextLine]:
+    """Recursively extracts text objects from a given parent object in a PDF document."""
+    objects = []
+
+    if isinstance(parent_object, LTTextLine):
+        objects.append(parent_object)
+    elif isinstance(parent_object, LTContainer):
+        for child in parent_object:
+            objects.extend(extract_text_objects(child))
+
+    return objects
```

Comment:
what's the compute cost of this function for a pdf page with lots of elements? This recursion could get expensive when there are deeply nested structures.